ibis/omniscidb/client.py

```diff
@@ -498,6 +508,17 @@ def _build_ast(self, expr, context):
         result = build_ast(expr, context)
         return result
 
+    def _check_execution_type(self, execution_type: int):
```

Comment:
i think the type hint here should be `ExecutionType`

Reply:
@toryhaavik I was thinking mabye a better solution for this would be a parameter as string instead of Enum. For example, pandas use some parameters as string instead of Enum.

also it is more practical to pass just a string than import a enum to use that

what is your thought about that?

Reply:
that's okay by me, but i'd still prefer to have an internal enum and cast the user's input to the enum. just want to stay as strongly typed inside the system as we can.

Reply:
I am waiting for some feedback from OmniSciDB team .. it seems there is new parameter to be used that can change the flow here.